### ISSUE TITLE
ipmi-config metrics

### ISSUE THREAD

**Issue by robbat2** [prometheus-community/ipmi_exporter#165]
I'm looking at a fleet with `ipmi_exporter` running locally on the nodes, and I'd like to export some of the `ipmi-config` values via labels.

```
ipmi_config_info{lan_ip_address="...",bmc_mac_address="..."} 1
```
Would this be accepted as a PR?

**Comment by bitfehler** [comment]
Yes, sounds good. It would become a new collector. Let me know if you need any pointers.
